src/esthri/Cargo.toml

```diff
@@ -37,13 +37,13 @@ name = "esthri"
 path = "src/lib.rs"
 
 [features]
-default = [ "rustls", "cli", "http_server", "blocking", ]
+default = [ "rustls", "cli", "http_server", "blocking", "tokio",]
```

Comment:
I think serde and async-stream also need to be marked as non-optional. I think originally they were only used for the server but they are both used in lib.rs now

Reply:
🛠️